### ISSUE TITLE
Sync related site URL

### ISSUE THREAD

**Issue by robinsm** [pbs/pbs-media-manager-craft-plugin#9]
The video below shows some more context around this issue of syncing the related site URL into a craft show entry.  The 3 toggle fields don't relate to this issue specifically (instead they are in https://github.com/pbs/pbs-media-manager-craft-plugin/issues/10), but are included for context.

Syncing this field would be a good start to making a new iteration of the media manager plugin for Ken Burns.

<img width="1445" alt="Cursor" src="https://user-images.githubusercontent.com/4096355/234785944-80404f32-cd2d-4767-8428-ac4785e9e11b.png">

https://user-images.githubusercontent.com/4096355/234785371-10c35e4f-9ed8-4f2a-af1d-16ae8e54734f.mp4

Full length video [here](https://drive.google.com/drive/u/3/folders/1nuXXTIFETjCVYrPWRBsPlPAZi3F6wxXz)

**Comment by andrewmenich** [comment]
Notes from the video:

"Stream with PBS Passport" lightswitch can be determined by whether the show is in Passport (FVOD, AKA Free Video on Demand) or not. I assume there's a flag or something sent back from the console API request that we can tap into

"Available to Purchase" lightswitch can be determined by the presence of various shop links sent back by the console API request, e.g. iTunes, Amazon, iOS, etc. under Content > Show Links

Related Site Url is sent back in the console request along with the purchase links and is (presumably) titled "Producer Site"

**Comment by andrewmenich** [comment]
The logic for this is quite simple and has been added to the KB repo here: https://github.com/pbs/pbs-media-manager-craft-kenburns/commit/fabcf579e93b512229f161a75f32038b5e8d8571

The reason it has not yet been merged is because in order to connect the data from the API to the Show entries, you have to update the plugin settings to indicate that a field in Craft is related to the data from the API. However, the plugin settings also have a custom-built field layout editor for both Media and Show sections. For some reason, the plugin settings do not pickup any changes to those sections made outside of the plugin settings page and get overridden whenever plugin settings are updated. It will take some time to untangle this. In the Craft 4 version of the plugin, we did away with the custom field layout altogether and perhaps this is what we should do here. It does not any benefit other than quickly setting up necessary fields when a project is first started. 

**TL;DR the plugin settings page is overwriting the Show section field layout, which results in unwanted and unintended. A solution needs established before we can merge this.**

**Comment by andrewmenich** [comment]
This logic has been added in [KB v3.3.3](https://github.com/pbs/pbs-media-manager-craft-kenburns/releases/tag/v3.3.3) and is on the KB Develop site for review. The issue listed above has been addressed via removing the plugin's handling of Show and Media section field layouts.

**Comment by SamLaneyPBS** [comment]
This is working!

**Comment by smaylath** [comment]
deployed to preprod & prod & completed 12.14.23
